presto-spi/src/main/java/com/facebook/presto/spi/block/SliceArrayBlock.java

```diff
@@ -177,7 +178,7 @@ public String toString()
 
     public static int getSliceArraySizeInBytes(Slice[] values)
     {
-        long sizeInBytes = SizeOf.sizeOf(values);
+        long sizeInBytes = values.length * (long) ARRAY_OBJECT_INDEX_SCALE;
```

Comment:
This removes the object overhead for the `Slice[]`, but shouldn't we include the object overhead for all of the Slice objects?  For other blocks we include the cost of offsets and nulls.